filer/migrations/0006_auto_20160623_1627.py

```diff
@@ -1,7 +1,4 @@
-# -*- coding: utf-8 -*-
 # Generated by Django 1.9.7 on 2016-06-23 20:27
-from __future__ import unicode_literals
-
 import django.db.models.deletion
```

Comment:
Change needed for 3.1 https://github.com/divio/django-filer/pull/1202/files#diff-8d2f6449635dadfd7a030ac92dfb1fd5R19